client/src/js/components/bhSoldOut/bhSoldOut.js

```diff
@@ -0,0 +1,48 @@
+angular.module('bhima.components')
+  .component('bhSoldOut', {
```

Comment:
Can you add the word "stock" to this?  "bhSoldOutStock"?  It would make it easier to look up.

Reply:
Good idea